CODEOWNERS

```diff
@@ -355,6 +355,10 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 # Lua cluster specifier
 /*/extensions/router/cluster_specifiers/lua @StarryVae @wbpcode
 
+# Outlier detection monitors
+/*/extensions/extensions/outlier_detection_monitors/common @cpakulski @nezdolik
+/*/extensions/extensions/outlier_detection_monitors/consecutive_errors @cpakulski @nezdolik
```

Comment:
Not sure should me be added to the list as initial maintainer reviewer?

Reply:
Great idea! Added you as well.